jsapp/js/components/map.es6

```diff
@@ -180,20 +192,46 @@ export class FormMap extends React.Component {
             iconSize: [20, 20],
           });
         }
-        prepPoints.push(L.marker(item._geolocation, {icon: icon, sId: item._id}));
+        prepPoints.push(
+          L.marker(item._geolocation, {
+            icon: icon, 
+            sId: item._id, 
+            typeId: viewby && mapMarkers ? mapMarkers[itemId].id : null
```

Comment:
IIUC this mirrors the if statement from line 175(187), so if the mentioned line will change, this could potentially be a bug